From a5f97ae1b99063383d8f45168125b34232b91faf Mon Sep 17 00:00:00 2001
From: Adrian Vondendriesch <adrian.vondendriesch@credativ.de>
Date: Tue, 17 Jan 2017 15:26:45 +0100
Subject: [PATCH] doc: document watchdog_device parameter

Commit 8d8d4a936ab73d8449a3574f969b17a90ef9428e introduced the
configuration parameter resources.watchdog_device. This commit
introduces the resources section and watchdog_device parameter in
corosync.conf.5.

Signed-off-by: Adrian Vondendriesch <adrian.vondendriesch@credativ.de>
Reviewed-by: Jan Friesse <jfriesse@redhat.com>
(cherry picked from commit 16ca44ccdc4f3ab051c866b15d948d44bf4ee020)
---
 man/corosync.conf.5 |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/man/corosync.conf.5 b/man/corosync.conf.5
index 55b21f2..546400e 100644
--- a/man/corosync.conf.5
+++ b/man/corosync.conf.5
@@ -60,6 +60,9 @@ This top level directive contains configuration options for nodes in cluster.
 .TP
 qb { }
 This top level directive contains configuration options related to libqb.
+.TP
+resources { }
+This top level directive contains configuration options for resources.
 
 .PP
 .PP
@@ -680,6 +683,17 @@ Native means one of shm or socket, depending on what is supported by OS. On syst
 with support for both, SHM is selected. SHM is generally faster, but need to allocate
 ring buffer file in /dev/shm.
 
+.PP
+Within the
+.B resources
+directive it is possible to specify options for resources.
+
+Possible option is:
+.TP
+watchdog_device
+This configuration option controls which watchdog device corosync try to bind to.
+The default value is /dev/watchdog. The value "no" disables the watchdog feature.
+
 .SH "FILES"
 .TP
 /etc/corosync/corosync.conf
-- 
1.7.1

